doc/classes/AnimatedSprite2D.xml

```diff
@@ -46,31 +75,44 @@
 		<member name="frame" type="int" setter="set_frame" getter="get_frame" default="0">
 			The displayed animation frame's index.
 		</member>
+		<member name="frame_progress" type="float" setter="set_frame_progress" getter="get_frame_progress" default="0.0">
+			The progress value between [code]0.0[/code] and [code]1.0[/code] until the current frame transitions to the next frame. If the animation is playing backwards, the value transitions from [code]1.0[/code] to [code]0.0[/code].
+		</member>
 		<member name="frames" type="SpriteFrames" setter="set_sprite_frames" getter="get_sprite_frames">
 			The [SpriteFrames] resource containing the animation(s). Allows you the option to load, edit, clear, make unique and save the states of the [SpriteFrames] resource.
 		</member>
 		<member name="offset" type="Vector2" setter="set_offset" getter="get_offset" default="Vector2(0, 0)">
 			The texture's drawing offset.
 		</member>
-		<member name="playing" type="bool" setter="set_playing" getter="is_playing" default="false">
-			If [code]true[/code], the [member animation] is currently playing. Setting this property to [code]false[/code] pauses the current animation. Use [method stop] to stop the animation at the current frame instead.
-			[b]Note:[/b] Unlike [method stop], changing this property to [code]false[/code] preserves the current frame's elapsed time and the [code]backwards[/code] flag of the current [member animation] (if it was previously set by [method play]).
-			[b]Note:[/b] After a non-looping animation finishes, the property still remains [code]true[/code].
-		</member>
-		<member name="speed_scale" type="float" setter="set_speed_scale" getter="get_speed_scale" default="1.0">
-			The animation speed is multiplied by this value. If set to a negative value, the animation is played in reverse. If set to [code]0[/code], the animation is paused, preserving the current frame's elapsed time.
+		<member name="playback_speed" type="float" setter="set_speed_scale" getter="get_speed_scale" default="1.0">
+			The speed scaling ratio. For example, if this value is 1, then the animation plays at normal speed. If it's 0.5, then it plays at half speed. If it's 2, then it plays at double speed.
```

Comment:
`playback_speed` and `custom_speed` can both be negative? I think this is confusing and overcomplicated, given that there is also animation FPS (it can't be negative though).

Reply:
Yes. But it's the same with AnimationPlayer. And you can get the multiplied speed with get_playing_speed(). The direction of the animation can be determined correctly by whether it is positive or negative. And it should be simpler than the backward flag.

Reply:
Perhaps it's worth mentioning that `speed_scale` can be negative?